target_chains/solana/sdk/js/pyth_solana_receiver/src/PythSolanaReceiver.ts

```diff
@@ -38,8 +39,14 @@ import {
 } from "@pythnetwork/solana-utils";
 import { PriorityFeeConfig } from "@pythnetwork/solana-utils/lib/transaction";
 
-export const DEFAULT_TREASURY_ID = 0;
-
+/**
+ * A class to interact with the Pyth Solana Receiver program.
+ *
+ * This class provides helpful methods to:
+ * - Post price updates from Pythnet to the Pyth Solana Receiver program
```

Comment:
suggest "from Pyth" not "Pythnet". Pythnet feels like an implementation detail

Reply:
Fixed